src/ext.js

```diff
@@ -20,6 +20,7 @@ export default function ext(/* env */) {
                   component: 'string',
                   ref: 'style.label',
                   translation: 'Label',
+                  placeholderTranslation: 'My button',
```

Comment:
Do we want a placeholder or a default value? Either way it should be possible to set the label empty so you get a blank button. Maybe solved if we allow expressions.

Reply:
I guess a default value would be easier then. But let's check with Christofer tomorrow!